templates/common/_base/ppc64le/files/disable-nic-tso.yaml

```diff
@@ -0,0 +1,10 @@
+filesystem: "root"
+mode: 0744
+path: "/etc/NetworkManager/dispatcher.d/pre-up.d/disable-nic-tso.sh"
+contents:
+  inline: |
+    #!/bin/bash
+    # This is a workaround for BZ#1816254.If the ibmveth driver is used, disable TSO
+    if lsmod|grep -q 'ibmveth'; then
```

Comment:
This could also live in RHCOS if we just did:

```
if "$(arch)" != "ppc64le"; then
  exit 0
fi
```
right?

Reply:
this would be great

Reply:
True. I'd be good with that myself especially if we don't foresee a need to have a multiarch aware MCO.